### PR TITLE
Suggest fig/cache-util

### PR DESCRIPTION
**Pull request by michaelcullum**

### CODE REVIEW

**Comment by michaelcullum** [comment]
@php-fig/psr-6

**Comment by chalasr** [comment]
👍

**Comment by Crell** [comment]
We should probably do the same for the other -util packages then.

**Comment by michaelcullum** [comment]
Yep. But this is the only util package other than link and I'd rather leave that as is until its passed.

**Comment by dragoonis** [comment]
Happy with this. Would you like me to merge now ? @michaelcullum

**Comment by k00ni** [comment]
What is the status here?

**Comment by Nyholm** [comment]
4 years later. 

No, I dont think this is needed. It is a great suggestion for an implementing library, but not for anyone else installing this package. 

👎

**Review by ashnazg — COMMENTED** [review]
For consistency with other `util suggest`'s verbiage, I'd remove the word `caching` here.

File: composer.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
             "Psr\\Cache\\": "src/"
         }
     },
+    "suggest": {
+        "fig/cache-util": "Provides some useful PSR-6 caching utilities"
```
Comment (Crell) [inline composer.json]:
```suggestion
        "fig/cache-util": "Provides some useful PSR-6 utilities"
```

**Review by ashnazg — APPROVED** [review]

**Comment by Nyholm** [comment]
Why would you want to suggest a package this package to EVERY psr6 user?

From its readme:

> While not useful for actual use, it serves as a demo-implementation as well as a debugging tool.

What will the average user benefit from this?

**Comment by Crell** [comment]
For the typical casual user of a cache library, probably not much.  However, `suggests` in my experience tends to list all kinds of things I never have an interest in. :smile: It tends to get used as a "see also."

**Comment by Nyholm** [comment]
> However, suggests in my experience tends to list all kinds of things I never have an interest in.

So why would you pile on to this list?

**Pull request merged**